infrastructure/etc/hosts

```diff
@@ -1,6 +0,0 @@
-# vRA IaaC START SECTION
```

Comment:
I see we are removing this file, are things working without it? is it the "extra_hosts" property the one that replaces hosts configuration (just note, i am asking because I don't have a lot of knowledge in this area and not as a change request for the pr)

Reply:
Yeah, we don't need this. I initially tried getting it to work, but never could. It appeared to be obsolete even initially, since we are not using the thus defined hosts, but instead the base infra.corp.local. Docker compose configures internal image-to-image hosts and addresses and on the host machine we only need the infra.corp.local 127.0.0.0 binding.

Reply:
I see thanks.